src/rebar3_ex_doc.erl

```diff
@@ -17,6 +17,7 @@
 
 -define(DEFAULT_DOC_DIR, "doc").
 -define(DEFAULT_DOC_LANG, "en").
+-define(DEFAULT_MERMAID_VSN, "latest").
```

Comment:
```suggestion
-define(DEFAULT_MERMAID_VSN, "10.2.4").
```

Reply:
You were right the first time 😄

Reply:
Ok.

Reply:
I'm force pushing to the prior commit. Fwiw, my change referred to this comment of yours: "I wonder if we should refer to the latest?"...

Reply:
This change is now reversed.

Reply:
Yeah, and this is why going slow even on a trivial change is valuable. An idea might seem good at the time, but falls apart on a follow up review or post-refelection. 

As stated, you were right the first time ;)